validators/types_unix.go

```diff
@@ -50,6 +50,8 @@ var DefaultSysSpec = SysSpec{
 			{Name: "NETFILTER_XT_TARGET_REDIRECT", Aliases: []string{"IP_NF_TARGET_REDIRECT"}},
 			{Name: "NETFILTER_XT_MATCH_COMMENT"},
 			{Name: "FAIR_GROUP_SCHED"},
+			{Name: "SECCOMP"},
+			{Name: "SECCOMP_FILTER"},
```

Comment:
hi, given we are adding this to "required", do we know if there are any popular distributions used for k8s that have this config explicitly disabled?

Reply:
I don't have the list of kernel configs of popular distributions although I have never heard to change the kernel config of seccomp from `y` to `n`.
I think some embedded systems may change the default to disable seccomp to reduce the impact of performance.

At first, I thought seccomp was one of requirements in the current Kubernetes.
But SeccompProfile is the optional currently and Kubernetes can launch the container without seccomp at least.
As of now, I think it is ok to change this check from `Required` to `Optional`.

Reply:
In addition, the seccomp e2e test is not included in the conformance check.
I'm sorry I should check it at first.

Reply:
I updated this check from `Required` to `Optional`.